### DISCUSSION TITLE
Inconsistent structure of reports when running tests as xUnit tests

### DISCUSSION THREAD

**Discussion by andreycha** [VIP-Logic/NBomber#657] [Q&A]
Hi,

I'm using NBomber 5.4.1 and NBomber.Http 5.0.1 to test REST API. I have 4 xUnit load tests with 1 scenario in every test. If I run all the tests from Visual Studio, `reports` folder is created in test project's `bin` directory and it contains 4 subfolders (1 per test run), e.g.:

![image](https://github.com/PragmaticFlow/NBomber/assets/1696046/d3262384-897f-4364-b073-e0f23abaca1b)

However, if I run tests from command line with `dotnet test`, I see that subfolders per test run are not created anymore and report files are written right into the `reports` folder, thus overwriting each other (last one wins).

Workaround: specify report folder like `.WithReportFolder($"reports\{testName}")`, so that it always puts test's report into separate folders.

Any advice on how to have subfolder per test run when using `dotnet test` without workaround? Is there anything I might be missing in my test setup or anything that can be adjusted/fixed on NBomber side?

Thank you.

**Comment by AntyaDev** [comment]
To be honest, I believe that every time when nBomber starts execution, it uses a unique sessionId, which should force NBomber to use a different folder for each new run.

I need to verify it.
